destination/destination_integration_test.go

```diff
@@ -0,0 +1,275 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/google/uuid"
+	"github.com/matryer/is"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/fault"
+)
+
+func TestDestination_Integration_Insert(t *testing.T) {
+	integrationTest(t)
+
+	is := is.New(t)
+	openAIKey := os.Getenv("OPENAI_APIKEY")
+	is.True(openAIKey != "") // expected OPENAI_APIKEY to be set
```

Comment:
hmmm we usually just skip the test with a log saying that this env var should be set, but I don't think we should fail

Reply:
You have a good point there. What made me do this is that otherwise we'll probably never know this test is failing for a PR, and there's nothing forcing us to run integration tests locally.

Reply:
oh didn't notice that you have the `OPENAI_APIKEY` added to the repo secrets! sure makes sense